### ISSUE TITLE
Table or colums in table disrupts the layout in outer table.

### ISSUE THREAD

**Issue by lech206** [bpampuch/pdfmake#2189]
Why layout in second and third table is disrupt?

```javascript
var dd = {
	content: [
      {
        table: {
          widths: ['50%', '50%'],
          body: [
            [
              {rowSpan: 4,
                stack: [
                  'text1',
                ],
              },
              'text2'
            ],
            ['', 'text3'],
            ['', 'text4'],
            ['', 'text5'],
            ['', 'text6']
          ],
        },
      },
      '-',
      {
        table: {
          widths: ['50%', '50%'],
          body: [
            [
              {rowSpan: 4,
                  table:{body:[['tcol1','tcol2']]}
              },
              'text2'
            ],
            ['', 'text3'],
            ['', 'text4'],
            ['', 'text5'],
            ['', 'text6']
          ],
        },
      },
      '-',
      {
        table: {
          widths: ['50%', '50%'],
          body: [
            [
              {rowSpan: 4,
                  columns:['col1','col2']
              },
              'text2'
            ],
            ['', 'text3'],
            ['', 'text4'],
            ['', 'text5'],
            ['', 'text6']
          ],
        },
      },
    ]
}
```
![image](https://user-images.githubusercontent.com/16007250/107931824-1b077200-6f7d-11eb-89b4-bed0b9cf94a4.png)

**Comment by mjomble** [comment]
The same issue was reported in #2069 with a simpler example

**Comment by liborm85** [comment]
Fixed by PR https://github.com/bpampuch/pdfmake/pull/2764.
Released in version 0.2.11.
